gpgwrapper/listprivateparser_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 Paul Furley and Ian Drysdale
```

Comment:
Why list*private*parser and not list*secret*parser ?

Reply:
Good catch - we call these _secret_ keys!